core/src/main/java/com/orange/ouds/core/component/OudsControlItem.kt

```diff
@@ -269,6 +276,38 @@ private fun LeadingTrailingBox(content: @Composable () -> Unit) {
     }
 }
 
+@Composable
+private fun ErrorDescriptionText(text: String) {
+    with(OudsTheme.componentsTokens.controlItem) {
+        Text(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = spacePaddingBlockTopErrorText.value)
```

Comment:
Figma indicates `ouds/💠_control/text-input/space/padding-block/top-helper-text` but I suppose the one you used is correct (I think I remember a discussion about this token) and this has not been updated on Figma, right?